packages/blade/fonts.css

```diff
@@ -0,0 +1,170 @@
+@font-face {
+  font-family: 'Tasa Orbiter';
+  src: url('./fonts/tasa-orbiter-vf.woff2') format('woff2-variations');
+  font-weight: 125 950;
+  font-stretch: 75% 125%;
+  font-style: normal;
+  font-display: swap;
+}
+
+/* inter-cyrillic-ext-400-normal */
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  font-display: swap;
+  font-weight: 400;
+  src: url(./fonts/Inter/inter-cyrillic-ext-400-normal.woff2) format('woff2'),
+    url(./fonts/Inter/inter-cyrillic-ext-400-normal.woff) format('woff');
+  unicode-range: U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
+}
+
+/* inter-cyrillic-400-normal */
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  font-display: swap;
+  font-weight: 400;
+  src: url(./fonts/Inter/inter-cyrillic-400-normal.woff2) format('woff2'),
+    url(./fonts/Inter/inter-cyrillic-400-normal.woff) format('woff');
```

Comment:
why do we need woff2 and woff both is it like a fallback for browsers that don't support woff2?

Reply:
Yup its a fallback for browsers